smsparkbuild/py39/Pipfile

```diff
@@ -0,0 +1,38 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+tenacity = "==5.1.4"
+psutil = "==5.7.0"
+click = "==8.0.0"
+watchdog = "==0.10.3"
+waitress = "==2.1.1"
```

Comment:
From online discussion: check and update external dependencies (ex, update boto3 to latest version)

Reply:
yep updated.